Packages/Application/Neos.NeosIo.Event/Resources/Private/TypoScript/NodeTypes/Event.ts2

```diff
@@ -0,0 +1,45 @@
+/**
+* NodeTypes around Events
+*/
+
+
+prototype(Neos.NeosIo.Event:Event.Document) < prototype(Neos.NeosIo:LandingPage.Document) {
+	body {
+		templatePath = 'resource://Neos.NeosIo.Event/Private/Templates/Page/Event.Landingpage.html'
+		content {
+			main = Neos.NeosIo.Event:Event
+		}
+		logo = ${q(site).property('eventLogo')}
```

Comment:
It should work due to this:
@context.site = ${q(node).closest('[instanceof Neos.NeosIo.Event:Event]').get(0) ? q(node).closest('[instanceof Neos.NeosIo.Event:Event]').get(0) : site}